packages/components/src/TreeView/TreeViewItem/TreeViewItem.component.js

```diff
@@ -213,6 +214,7 @@ class TreeViewItem extends React.Component {
 			actions,
 			icon,
 			counter = children.length,
+			disabled,
```

Comment:
`disabled` must be passed to the Action, isn't it?

Reply:
I thought about it, disabling all actions limits our flexibility.  We can set actions to be disabled using action props, and with this approach, disabled list items may contain other clickable actions (info, warning etc) that may be better for the user experience.

Reply:
The onselect action is still fired on the demo so I don t understand it s only visual disabled right ?

Reply:
Yes, you are right. We can also skip onSelect callback for disabled items. But onToggle should be available for the user. We use it on plan rerun dialog, and the user should be able to see what's inside disabled steps.

Reply:
Using the keyboard I can still fire the onSelect on the demo, so we must consider modifying the other `onKey*`handlers.

Another thing, we need to get the info that this button is actually `[aria-disabled=true]` if we want to manage it manually!

Reply:
good catch @frassinier !